static/src/javascripts/projects/common/utils/steady-page.js

```diff
@@ -0,0 +1,188 @@
+/*
+ Utility: steady-insert.js
+ Insert components into web-pages without jumping them around by
+ moving the current scroll position by the same distance as the height of the
+ inserted element.
+ Note: Inserted elements must have a fixed height when rendered
+ (i.e: no changing the height by loading an image, lazy-loading content etc)
+ */
+
+define([
+    'common/utils/fastdom-promise',
+    'Promise',
+    'lodash/objects/assign',
+    'common/utils/Queue',
```

Comment:
even though this is a constructor I think we should only use lowercase for consistency. [require is case sensitive](https://github.com/requirejs/requirejs/issues/270) so let's not add a variation for this one module. I'd make the filename lowercase as well.